neptune/internal/utils/json_file_splitter.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+from json import JSONDecodeError
+from typing import Optional
+
+
+class JsonFileSplitter:
+
+    # TODO: experiment with larger buffer sizes
+    BUFFER_SIZE = 4096
```

Comment:
Out of curiosity: why this size?

Reply:
It is default internal buffer size in splitstream library (https://github.com/rickardp/splitstream) which eventually I dicided not to use. But 4kb seems to be good guess.